docs/wfm-stitching.ipynb

```diff
@@ -0,0 +1,619 @@
+    "        close=sc.array(\n",
+    "            dims=[\"cutout\"],\n",
+    "            values=[134.6, 190.06, 237.01, 280.88, 323.56, 373.76],\n",
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        phase=12.27 * deg,\n",
+    "        distance=15.9 * meter,\n",
+    "        name=\"Frame-overlap 2\",\n",
+    "    ),\n",
+    "    tof.Chopper(\n",
+    "        frequency=7 * Hz,\n",
+    "        open=sc.array(\n",
+    "            dims=[\"cutout\"],\n",
+    "            values=[30.0],\n",
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        close=sc.array(\n",
+    "            dims=[\"cutout\"],\n",
+    "            values=[140.0],\n",
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        phase=0 * deg,\n",
+    "        distance=22 * meter,\n",
+    "        name=\"Pulse-overlap\",\n",
+    "    ),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bd6e1399-7f05-4d5c-83e3-7d249d9e0a61",
+   "metadata": {},
+   "source": [
+    "## Detector set-up\n",
+    "\n",
+    "We add a detector 32 meters from the source."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a377e75c-51a5-4994-af9c-92139d27bee3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "detectors = [\n",
+    "    tof.Detector(distance=32.0 * meter, name=\"detector\"),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fcb52d39-56d6-4cfb-8bcd-be9f45f4cc70",
+   "metadata": {},
+   "source": [
+    "## Find WFM frame edges\n",
+    "\n",
+    "To compute the frame edges, we use one of the openings in the WFM choppers are a time,\n",
+    "run the `tof` simulation, and find the min and max tof of the neutrons that make it through the chopper cascade."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20597dbd-5204-49fc-8867-9a9ba73fd8de",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nframes = len(choppers[0].open)\n",
+    "results = []\n",
+    "\n",
+    "print(\"Computing frames\")\n",
+    "\n",
+    "# Run the model with a single frame at a time\n",
+    "for i in range(nframes):\n",
+    "    wfm_choppers = [\n",
+    "        tof.Chopper(\n",
+    "            frequency=choppers[j].frequency,\n",
+    "            open=choppers[j].open[i : i + 1],\n",
+    "            close=choppers[j].close[i : i + 1],\n",
+    "            phase=choppers[j].phase,\n",
+    "            distance=choppers[j].distance,\n",
+    "            name=choppers[j].name,\n",
+    "        )\n",
+    "        for j in (0, 1)\n",
+    "    ]\n",
+    "    new_choppers = wfm_choppers + choppers[2:]\n",
+    "    model = tof.Model(source=source, choppers=new_choppers, detectors=detectors)\n",
+    "    res = model.run()\n",
+    "    results.append(res)\n",
+    "\n",
+    "invalid_frames = True\n",
+    "fact = 0.01\n",
+    "while invalid_frames:\n",
+    "    print(f\"Searching for bounds: threshold={fact}\")\n",
```

Comment:
This part renders a bit funny.
Should it be rounded up?

![image](https://github.com/scipp/tof/assets/17974113/ddc1b5ad-0408-427d-9ce7-8e4a4ec94024)